_docs/controller/configuration-options.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title: Configuration
+permalink: /docs/controller/controller-configuration-options/
+category: Controller
+order: 3
+---
+
+The following instructions apply to both Docker-based and Kubernetes-based
+installations. The Amalgam8 Controller supports a number of configuration
+options, most of which are set through environment variables. The environment
+variables can be set via command line flags as well.
+
+The following environment variables are available. All of them are optional.
+
+| Environment Key | Flag Name                   | Description | Default Value |
+|:----------------|:----------------------------|:------------|:--------------|
+| A8_API_PORT | --api_port | API port | 8080 |
+| A8_CONTROL_TOKEN | --control_token | Controller API authentication token | ABCDEFGHIJKLMNOP |
+| A8_ENCRYPTION_KEY | --encryption_key | secret key | abcdefghijklmnop |
+| A8_DATABASE_TYPE |  --database_type |	database type | memory |
+| A8_DATABASE_USERNAME | --database_username | database username | |
+| A8_DATABASE_PASSWORD | --database_password | database password | |
+| A8_DATABASE_HOST | --database_host | database host | |
+| A8_LOG_LEVEL | --log_level | logging level (debug, info, warn, error, fatal, panic) | info |
+| A8_AUTH_MODE | --auth_mode | Authentication modes. Supported values are: 'trusted', 'jwt'" | |
+| A8_JWT_SECRET | --jwt_secret | Secret key for JWT authentication | |
```

Comment:
How does this work with multiple tenants? Might be useful to add a subsection after the persistent storage, on something like
"Setting up Controller in multi-tenant mode". And talk about the trusted, jwt auth mode, how each tenant (with multiple sidecars) will use their tokens, etc.